content/client-server-api/modules/receipts.md

```diff
@@ -137,16 +137,22 @@ either a thread root's event ID or `main` for the main timeline.
 
 Threading introduces a concept of multiple conversations being held in the same
 room and thus deserve their own read receipts and notification counts. An event is
-considered to be "in a thread" if it meets any of the following criteria:
-* It has a `rel_type` of `m.thread`.
-* It has child events with a `rel_type` of `m.thread` (in which case it'd be the
-  thread root).
-* Following the event relationships, it has a parent event which qualifies for
-  one of the above. Implementations should not recurse infinitely, though: a
-  maximum of 3 hops is recommended to cover indirect relationships.
-
-Events not in a thread but still in the room are considered to be part of the
-"main timeline", or a special thread with an ID of `main`.
+considered to be "in a thread" if:
+
+* It has a rel_type of m.thread, or
+* It has a parent event with this rel_type, or a parent of a parent, or further
+  up the chain of relations. (Implementations should not recurse to arbitrary
+  depth: a maximum of 3 hops is recommended to cover indirect relationships.)
+
+Events not in a thread but still in the room are considered to be in the "main
+timeline". Threaded receipts for the main timeine use a special thread ID of
+`main`.
```

Comment:
It isn't just about receipts, it is also about notification counts, etc. which is why it is noted to have a special ID.

Reply:
That would be fine. If a future use of it comes along then the phrasing can be updated. 👍